web/src/condense.ts

```diff
@@ -18,38 +19,47 @@ This library implements two related, similar concepts:
 
 */
 
-function show_more_link($row) {
+function show_more_link($row: JQuery): void {
     $row.find(".message_condenser").hide();
     $row.find(".message_expander").show();
 }
 
-function show_condense_link($row) {
+function show_condense_link($row: JQuery): void {
     $row.find(".message_expander").hide();
     $row.find(".message_condenser").show();
 }
 
-function condense_row($row) {
+function condense_row($row: JQuery): void {
     const $content = $row.find(".message_content");
     $content.addClass("condensed");
     show_more_link($row);
 }
 
-function uncondense_row($row) {
+function uncondense_row($row: JQuery): void {
     const $content = $row.find(".message_content");
     $content.removeClass("condensed");
     show_condense_link($row);
 }
 
-export function uncollapse($row) {
+export function uncollapse($row: JQuery): void {
     assert(message_lists.current !== undefined);
+    assert(message_lists.home !== undefined);
+    const id = rows.id($row);
+    if (id === undefined) {
+        return;
+    }
 
     // Uncollapse a message, restoring the condensed message "Show more" or
     // "Show less" button if necessary.
-    const message = message_lists.current.get(rows.id($row));
```

Comment:
In #29159 I’ve removed the `undefined` return from `rows.id`, so this no longer needs to be changed.